pages/open_cas_linux/guide_io_classification.md

```diff
@@ -28,7 +28,7 @@ below summarizes the configurable fields in this IO classification file.
 
 | **Field**         | **Description**                                                                                                                                                                                                                                       |
 |-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| IO class id       | Unique ID for the IO class. Values in this field are fixed and should not be modified. (eg. ID for Metadata must always be 1)                                                                                                                         |
+| IO class id       | Unique ID for the IO class. (NOTE: ID for `unclassified` must always be 0)                                                                                                                                                                            |
```

Comment:
\`unclassified\` in back quotes results in displaying the word in a frame `unclassified` - is that intended?
NOTE: The ID

Reply:
If it's intended then I think it's inconsistent.. I'd imagine it would make sense to put all of the keywords/rule examples in those and that's the option I'd go for.